core/math/vector4.cpp

```diff
@@ -53,6 +87,16 @@ bool Vector4::is_normalized() const {
 	return Math::is_equal_approx(length_squared(), 1, (real_t)UNIT_EPSILON); // Use less epsilon.
 }
 
+real_t Vector4::distance_to(const Vector4 &p_to) const {
```

Comment:
I feels these are entirely useless, but if it makes you feel better I don't object to merging them.